app/models/taxon_facet.rb

```diff
@@ -0,0 +1,92 @@
+class TaxonFacet < DropdownSelectFacet
+  TOP_LEVEL_TAXON_KEY = 'level_one_taxon'.freeze
+  SECOND_LEVEL_TAXON_KEY = 'level_two_taxon'.freeze
+
+  def topics
```

Comment:
This is a `TaxonFacet` but it returns `topics`. I wonder whether we're leaking in too much context about how that GOV.UK taxonomy is structured. You need to know that the taxons are topics.

Maybe we could change the API to something like `level_one_values` and `level_two_values`?